packages/cache/types/index.d.ts

```diff
@@ -18,20 +18,20 @@ export interface StylisElement {
 export type StylisPluginCallback = (
   element: StylisElement,
   index: number,
-  children: Array<StylisElement>,
+  children: ReadonlyArray<StylisElement>,
   callback: StylisPluginCallback
 ) => string | undefined
 
 export type StylisPlugin = (
   element: StylisElement,
   index: number,
-  children: Array<StylisElement>,
+  children: ReadonlyArray<StylisElement>,
```

Comment:
Changing these to Readonly seem wrong? Aren't these arrays mutable?

Reply:
Yep, you are right 👍